ui/lib/components/TreeDiagram/Minimap.tsx

```diff
@@ -194,46 +153,44 @@ const Minimap = ({
     ;(brushRef as MutableRefObject<SVGElement>).current = _brushRef.current
   }, [brushRef])
 
+  const Trees = useMemo(
```

Comment:
How about updating `_Trees` to `MemorizedTrees` and 
putting it somewhere with SingleTree?